web/src/components/Storyblok/CardGridSection/CardGridSection.test.tsx

```diff
@@ -362,7 +362,49 @@ describe("CardGridSection", () => {
 		expect(cardList).not.toBeInTheDocument();
 		expect(cardListItems).not.toBeInTheDocument();
 	});
-
+	it("renders Testimonial content correctly with promobox", () => {
+		const mockTestimonialProps: CardGridSectionProps = {
+			blok: {
+				_uid: "5b2cf64a-dde9-4bd1-9abd-90ff4164d932",
+				columns: "3",
+				cards: [],
+				gridItems: [
+					{
+						_uid: "22efcc3f-27a1-49d3-bc70-06b75e6699c8",
+						image: {
+							id: 20714807,
+							alt: "",
+							name: "",
+							focus: "",
+							title: "",
+							source: "",
+							filename:
+								"https://a.storyblok.com/f/292509/698x418/64068e0d3b/severe-dust-mite-allerg.JPG",
+							copyright: "",
+							fieldtype: "asset",
+							meta_data: {},
+							is_external_url: false,
+						},
+						variant: "default",
+						component: "testimonialGridItem",
+						quoteName: "Quote Name",
+						quoteRole: "Quote Role",
+						quoteText: "Quote Text",
+						_editable:
+							'<!--#storyblok#{"name": "testimonialGridItem", "space": "292509", "uid": "22efcc3f-27a1-49d3-bc70-06b75e6699c8", "id": "628392114"}-->',
```

Comment:
We probably don't want to be committing our space ids into the public repo - is '_editable' necessary in the mock props ?